documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR.
+2. If the service has onboarded DPG before, please go to step 4. Otherwise, please go to step 3.
+3. Create branch with name `dpg/<prNumber>` in sdk repository, and add `autorest.md` or `README.md`. Then add a comment `/azp run` in swagger PR.
+   1. `<prNumber>` is the PR number of swagger PR.
+   2. About how to add `autorest.md` or `README.md`, please refer to [Add autorest configuration in sdk repository](./add-autorest-configuration-in-sdk-repository.md).
+4. Swagger PR CI generates SDK and ApiView automatically, and you can find them in swagger PR comments.
+5. Ask Arch Board to review the generated ApiView.
+6. If Arch Board approves the ApiView, go to step 10. Otherwise, go to step 7.
+7. Update Swagger if needed.
+8. Update `autorest.md` or `README.md` if needed. Here, if there is a branch named `dpg/<prNumber>` in sdk repository, please update it in the branch. Otherwise update it in main branch.
```

Comment:
If there is change for the autorest.md,  shall we ask them to always create  a branch named `dpg/<prNumber>` , i don't think people could update main branch directly.

Reply:
Yes, I change to ask them to create a new branch `dpg/<prNumber>` to update the configuration.